File: src/main/java/yunas/http/DefaultContentType.java
package yunas.http;

/**
 * DefaultContentType.
 *
 * @author yosuke kobayashi
 */
public enum DefaultContentType  {

    JSON("application/json"),
    XML("application/xml"),
    HTML("text/html"),
    PLAIN("text/plain");

    private String value;

    DefaultContentType(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: src/main/kotlin/yunas/http/DefaultContentType.kt
package yunas.http

/**
 * DefaultContentType.
 *
 * @author yosuke kobayashi
 */
enum class DefaultContentType private constructor(val value: String) {

    JSON("application/json"),
    XML("application/xml"),
    HTML("text/html"),
    PLAIN("text/plain")
}
